src/nodes/Table/TableView.vue

```diff
@@ -50,25 +71,60 @@ export default {
 			type: Function,
 			required: true,
 		},
+		node: {
+			type: Object,
+			required: true,
+		},
 	},
 	data() {
 		return {
 			isEditable: false,
+			isFocused: false,
 		}
 	},
 	beforeMount() {
 		this.isEditable = this.editor.isEditable
+		this.editor.on('selectionUpdate', ({ editor }) => {
+			const startOfCurrentNode = this.getPos()
+			const endOfCurrentNode = startOfCurrentNode + this.node.nodeSize
+			const { from, to } = editor.state.selection
+			this.isFocused = from >= startOfCurrentNode && to <= endOfCurrentNode
+		})
 		this.editor.on('update', ({ editor }) => {
 			this.isEditable = editor.isEditable
 		})
 	},
+	methods: {
+		addColumnAfter() {
+			const headerRowNode = this.node.firstChild
+			this.editor.chain()
+				.focus()
+				.setTextSelection(this.getPos() + headerRowNode.nodeSize - 1)
+				.addColumnAfter()
+				.run()
+		},
+		addRowAfter() {
+			const lastRowNode = this.node.lastChild
+			this.editor.chain()
+				.focus()
+				.setTextSelection(this.getPos() + this.node?.nodeSize - lastRowNode.nodeSize + 1)
+				.addRowAfter()
+				.run()
+		},
+	},
 }
 </script>
 
 <style scoped lang="scss">
 .table-wrapper {
 	position: relative;
 	overflow-x: auto;
+
+	&.focused {
```

Comment:
I'd expect the `&.focused` ruleset to be inside the `.table-add-column` and `.table-add-row` rulesets below, but that's rather nitpicking.

Reply:
This is the focused state of table. So, I think it should be inside `.table-wrapper`.